src/simularium/VisData.ts

```diff
@@ -389,6 +399,17 @@ class VisData {
         if (this.firstFrameTime === null) {
             this.firstFrameTime = frames.frameDataArray[0].time;
         }
+
+        if (this.frameDataCache.length > this.maxCacheLength) {
```

Comment:
Food for thought: how would you do this if you needed to limit the cache based on bytes?  For example "we want to cache up to 2GB of frames but no more".  The reason I ask is because 2000 frames of one trajectory could be wildly different in memory consumed than 2000 frames of a different trajectory.

Reply:
@meganrm just made this point in our 1:1 as well

I think that is definitely the next step, and then the length property might be totally discarded or only used for the case of user provided limit, or a minimal/no caching scenario.

We could table this PR on behalf of not using length at all and re-open with a memory based limit, but this felt like the easiest way to introduce the ability to cap the cache for use in `binding-sim`.